app/views/layout/layout.pug

```diff
@@ -1,7 +1,7 @@
 include ../blocks/mixins
 include ../blocks/header
 include ../blocks/footer
-block vars
+block title
```

Comment:
Thay đổi như này coi chừng ảnh hưởng tới những trang khác đấy
Nhưng tại sao phải đổi từ vars sang title nhỉ?

Reply:
à em đổi sang cho cho khác những cái block vars để tính toán ạ :D